server/graphql/types/src/types/activity_log.rs

```diff
@@ -187,6 +188,7 @@ impl ActivityLogNodeType {
             from::AssetLogReasonDeleted => to::AssetLogReasonDeleted,
             from::AssetPropertyCreated => to::AssetPropertyCreated,
             from::VaccineCourseCreated => to::VaccineCourseCreated,
+            from::VaccineCourseUpdated => to::VaccineCourseUpdated,
```

Comment:
I think we need to add this type for postgres right? Do postgres tests pass?

Reply:
[8e8e143](https://github.com/msupply-foundation/open-msupply/pull/4142/commits/8e8e143ded821d918642c4c77f166ec31e770397)